include/pref_reports.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of the QPxTool project.
+ * Copyright (C) 2010 Gennady "ShultZ" Kozlov <qpxtool@mail.ru>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file "COPYING" for the exact licensing terms.
+ *
+ */
+
+#ifndef PREF_REPORTS_H
+#define PREF_REPORTS_H
+
+#include "qpxsettings.hpp"
+#include <QWidget>
+
+class QGridLayout;
+class QHBoxLayout;
+class QVBoxLayout;
+
+class QGroupBox;
+class QLabel;
+class QLineEdit;
+class QPushButton;
+class QCheckBox;
+class QComboBox;
+class QSpinBox;
+
+class prefReports : public QWidget {
+  Q_OBJECT
+
+public:
+  prefReports(QPxSettings *iset, QWidget *p = 0, Qt::WindowFlags fl = 0);
```

Comment:
Codacy found an issue: [Class 'prefReports' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)